src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: myFont;
+  src: url(./fonts/Avenir.ttc);
+
+}
+
+body {
+  font-family: myFont, sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+  margin: 8px;
+  padding: 0;
+}
+
+.search {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  height: 68px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin: 20px auto 0;
+}
+
+.search:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search_big__line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  padding: 23px 0 24px 25px;
+}
+
+.search_big__icon {
+  display: inline-block;
+  width: 19px;
+  height: 19px;
+  padding-right: 18px;
+}
+
+.search__input {
```

Comment:
this should be our general class, in addition, there should be only a class for it that will slightly change the styles for the small search bar

Reply:
done